sql/core/src/test/resources/sql-tests/inputs/change-column.sql

```diff
@@ -0,0 +1,50 @@
+-- Create the origin table
+CREATE TABLE test_change(a Int, b String, c Int);
+CREATE VIEW test_view(a, b, c) AS
+SELECT * FROM VALUES (1, "one", 11), (null, "two", 22) AS testData(a, b, c);
```

Comment:
We have to make the column `a` nullable here, otherwise it can't be coverted to `StructField(a,IntegerType,true,{comment: newComment})`